Source/HTTP/httpcall.h

```diff
@@ -66,6 +66,9 @@ struct HC_CALL
     xbox::httpclient::HttpHeaders responseHeaders{};
     HCHttpCallResponseBodyWriteFunction responseBodyWriteFunction{ HC_CALL::ResponseBodyWrite };
     void* responseBodyWriteFunctionContext{ nullptr };
+    HCHttpCallResponseBodyWriteFunction temporaryResponseBodyWriteFunction{ HC_CALL::ResponseBodyWrite };
```

Comment:
Maybe the logic is a bit cleaner if you call this something like "clientResponseBodyWriteFunction"?  and I'd probably initialize to nullptr

Reply:
either way, add a quick comment describing why this is needed